shared/Cargo.toml

```diff
@@ -28,3 +27,6 @@ approx = "0.4"
 redis = { version = "0.19", default-features = false, optional = true }
 rand = "0.8"
 persy = { version = "0.11", optional = true }
+
+[dev-dependencies]
+anyhow = "1"
```

Comment:
This dependency is already in the main dependencies list?

Reply:
Uhm, yes.